### ISSUE TITLE
训练YOLOX_Nano，如果用4张卡，-b设置为32，training config中self.basic_lr_per_img = 0.01 / 64.0需要改为self.basic_lr_per_img = 0.01 / 32.0吗？

### ISSUE THREAD

**Issue by lixiangMindSpore** [Megvii-BaseDetection/YOLOX#974]
训练YOLOX_Nano，如果用4张卡，-b设置为32，training config中self.basic_lr_per_img = 0.01 / **64.0**需要改为self.basic_lr_per_img = 0.01 / **32.0**吗？还需要修改别的配置参数吗？
![image](https://user-images.githubusercontent.com/78945582/144952577-86694560-adc3-4a16-9126-2d477581f5cf.png)

**Comment by lixiangMindSpore** [comment]
> ### 训练YOLOX_Nano，如果用4张卡，-b设置为32，training config中self.basic_lr_per_img = 0.01 / **64.0**需要改为self.basic_lr_per_img = 0.01 / **32.0**吗？还需要修改别的配置参数吗？ ![image](https://user-images.githubusercontent.com/78945582/144952577-86694560-adc3-4a16-9126-2d477581f5cf.png)

**Comment by cyh767** [comment]
感觉不用。文章第二页里说

> We use a learning rate of lr×BatchSize/64 (linear scaling [8])

对应代码里yolox/core/trainer.py下的这一行

`self.lr_scheduler = self.exp.get_lr_scheduler(
            self.exp.basic_lr_per_img * self.args.batch_size, self.max_iter
        )`

所以这里的0.01/64.0就是乘到batch大小上的系数，不用随batch大小动的。个人观点，欢迎指正。

**Comment by Joker316701882** [comment]
@lixiangMindSpore Our code will automatically adjust the learning rate based on the batch size.
